spec/controllers/application_templates_controller_spec.rb

```diff
@@ -120,7 +121,7 @@
       end
       it 'redirects back' do
         submit
-        expect(response).to redirect_to :back
+        expect(response).to redirect_to 'http://test.host/redirect'
```

Comment:
On line 108:
`request.env['HTTP_REFERER'] = 'http://test.host/redirect'`
So yes. If you think it would make it clearer, on the next line I could totally
`@back = request.env['HTTP_REFERER']`